modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/resources/META-INF/resources/view_records.jsp

```diff
@@ -201,9 +201,9 @@ if (!ddlDisplayContext.isAdminPortlet()) {
 		deleteRecords: deleteRecords,
 	};
 
-	Liferay.componentReady('<%= randomNamespace + "ddlViewRecordsManagementToolbar" %>').then(function (
-		managementToolbar
-	) {
+	Liferay.componentReady(
+		'<%= randomNamespace + "ddlViewRecordsManagementToolbar" %>'
+	).then(function (managementToolbar) {
```

Comment:
Sneaking in some formatting fixes eh?

Reply:
Not sure :D

Reply:
I'm going to look the other way with this one... let it sneak in... like a ninja...